### DISCUSSION TITLE
Sort via prop

### DISCUSSION THREAD

**Discussion by kurtnz** [TanStack/table#2138] [General] [answered]
Can someone smarter than me help figure out how I can pass in a column ID for the table to be sorted by? 

I'm my particular case I have a <select> element sitting in a parent component, when this changes I'd like the table to be sorted via the value that is selected.

Any advise or a code sandbox would be greatly appreciated.

**Comment by gargroh** [comment, marked as answer]
This can help you here https://github.com/tannerlinsley/react-table/blob/master/src/plugin-hooks/useSortBy.js#L359
